packages/core/src/discovery.js

```diff
@@ -177,7 +185,7 @@ async function* captureSnapshotResources(page, snapshot, options) {
 
   // navigate to the url
   yield resizePage(snapshot.widths[0]);
-  yield page.goto(snapshot.url);
+  yield page.goto(snapshot.url, cookies);
```

Comment:
Why passing as separate argument, can't we pass as an option?

Reply:
This will break lot of flows where we use page.goto as it will now require cookies to be passed explicitly.